net-libs/libblkmaker/libblkmaker-0.6.0.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools multilib-minimal
+
+DESCRIPTION="C implementation of Bitcoin's getblocktemplate interface"
+HOMEPAGE="https://github.com/bitcoin/libblkmaker"
+LICENSE="MIT"
+
+SRC_URI="https://github.com/bitcoin/${PN}/archive/v${PV}.tar.gz -> ${P}-github.tgz"
+SLOT="0/7"
+KEYWORDS="~amd64 ~arm ~ppc ~ppc64 ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	>=dev-libs/jansson-2.0.0
+	dev-libs/libbase58
```

Comment:
```
 * abi_x86_32.x86: running multilib-minimal_abi_src_configure
...
...
checking for libbase58... no
configure: error: Package requirements (libbase58) were not met:

Package 'libbase58', required by 'virtual:world', not found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables libbase58_CFLAGS
and libbase58_LIBS to avoid the need to call pkg-config.
```

libbase58 is not multilib'd so it wouldn't work with a revdep that requires lib32 compatibility.